src/main/java/nc/students/ayaz/Comment.java

```diff
@@ -7,20 +7,20 @@
 
     private String author;
     private String text;
-    private Date date;
 
     Comment(String author, String text) {
         this.author = author;
         this.text = text;
-        this.date = Calendar.getInstance().getTime();
     }
 
-    private String getAuthor() {
-        return author;
-    }
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
&#x1F34E; **Issue**
Why cant we remove this row? If we have same objects then equals bellow will return true.